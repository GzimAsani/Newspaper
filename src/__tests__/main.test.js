import { BrowserRouter } from 'react-router-dom';
import { create } from 'react-test-renderer';
import Main from '../components/main';

test('renders correctly', () => {
  const Placeholder = () => (
    <BrowserRouter>
      <Main />
    </BrowserRouter>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
