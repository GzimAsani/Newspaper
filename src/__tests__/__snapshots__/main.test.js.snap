// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className=" text-center"
>
  <h1
    className="text-success"
  >
    Unnamed NewsPaper
  </h1>
  <img
    alt="Newspaper photo"
    className="img-fluid"
    src="newspaper.jpg"
  />
</div>
`;
